group_project_v2/project_api/api_implementation.py

```diff
@@ -273,11 +273,11 @@ def get_project_by_content_id(self, course_id, content_id):
             'course_id': course_id
         }
         response = self.send_request(GET, (PROJECTS_API,), query_params=query_params)
-        assert len(response) <= 1
+        assert len(response['results']) <= 1
```

Comment:
@ziafazal I would suggest moving `if not response` above this line and probably changing it to `if not response or not response['results']`